go.mod

```diff
@@ -0,0 +1,8 @@
+module github.com/jamietsao/aws-ssm-env
```

Comment:
Switched to go modules

Reply:
Oh hmm, I guess this should switch to to `gametimesf`.  How is this typically done via upstream PRs?  Should a merge from upstream first happen in a branch, then make these package path updates, then PR?  cc @ajanata

Reply:
After some research, if I understand correctly, GT's fork should keep the original module path of `jamietsao`, and in the repos that use this fork, the `replace` directive for go modules should be used:
https://github.com/golang/go/wiki/Modules#when-should-i-use-the-replace-directive
https://stackoverflow.com/a/56792766

Is that right?

Reply:
Yea that's seems like the cleanest way to do it. Otherwise it gets more difficult to bring in upstream changes like we're doing now without doing a little juggling to change the paths.

Reply:
I guess it makes sense to use `replace` there, I don't think we actually import this in more than a couple places